test/dlc_tests/conftest.py

```diff
@@ -176,17 +185,18 @@ def ec2_instance(
                 LOGGER.error(f"Failed to launch in {a_zone} with Error: {e}")
                 continue
     else:
-        instances = ec2_resource.create_instances(**params)
+        return None
+        # try:
+        #     instances = ec2_resource.create_instances(**params)
+        # except ClientError as e:
+        #     if ec2_instance_type == "p3dn.24xlarge" and "InsufficientInstanceCapacity" in str(e):
+        #         return None
```

Comment:
Could you add logging here?
Like "Failed to launch p3dn.24xlarge in {a_zone} because of insufficient instance capacity"

Reply:
Done.